src/styles/application/_copyField.scss

```diff
@@ -0,0 +1,36 @@
+.integr8ly-copy {
+  &-input {
+    &.false {
+      width: 200px;
+      text-overflow: ellipsis;
+    }
+    &.expanded {
+      background-color: $color-pf-black-200;
+    }
+  }
+  &-display {
+    width: 100%;
+    padding: 15px;
+    border: 1px solid $color-pf-black-400;
+    border-top-width: 0;
+    word-wrap: break-word;
+    resize: none;
+  }
+  & > .input-group {
+    display: flex;
+    .input-group-addon,
+    .input-group-btn {
+      width: auto;
+    }
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
@priley86 this can be moved right under the first declaration of `integr8ly-copy-input` - you place it right under the first `width` definition.

```
.integr8ly-copy {
  &-input {
    &.false {
      width: 200px;
      @media (min-width: 768px) {
        width: 400px;
      }
      text-overflow: ellipsis;
    }
```

Reply:
updated this, squashed commits, and rebased